fe/fe-core/src/main/java/com/starrocks/load/DeleteMgr.java

```diff
@@ -508,6 +508,11 @@ private boolean checkDelete(OlapTable table, List<Partition> partitions, List<Pr
                         "is not duplicate or column type is float or double.");
             }
 
+            if (column.getType().isComplexType()) {
+                throw new DdlException(
+                        "unsupported delete condition on Array/Map/Struct type column[" + columnName + "]");
```

Comment:
what about json/bitmap or other complex types?

Reply:
`bitmap/hll` is `Aggregate-Value` column, can't be delete condition. We can delele by `Json` type condition